app/main/api/handlers/getWallet.ts

```diff
@@ -123,9 +123,12 @@ async function searchWallet(
  * @param {data: JsonSerializable}
  * @returns {Promise<Wallet>}
  */
-async function parseWallet(data: JsonSerializable): Promise<Wallet> {
+async function parseWallet(data: JsonSerializable, system: SubSystems): Promise<Wallet> {
   try {
-    return asRuntimeType(data, Wallet)
+    const wallet = asRuntimeType(data, Wallet)
+    system.appStateManager.update({ wallet })
```

Comment:
I'm under the impression that now that we store the `wallet` in the [AppState](https://github.com/witnet/sheikah/pull/436/files#diff-033d58029e7e5f4e0726a62f0605b05c), the app state should also be updated with the wallet in the `encryptWallet`handler

Reply:
Just pointing out things here to address them somewhere, but I'm ok with merging and then taking them into account separately :ok_hand: